frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/setup.tsx

```diff
@@ -24,7 +24,12 @@ export const setupEmbedding = async ({
   hasEnterprisePlugins = false,
 }: SetupOpts) => {
   const returnedValue = await setup({
-    settings: [createMockSettingDefinition({ key: "enable-embedding" })],
+    settings: Object.entries(settingValues ?? {}).map(([key, value]) => {
```

Comment:
since we use `getSettings` selector, we need to populate setting value here as well.